src/components/ChattingInput/style.ts

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+
+export const ChattingInput = styled.div`
+  width: calc(100vw - 31.3125rem);
+  height: 5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const InputContainer = styled.div`
```

Comment:
```suggestion
export const InputContainer = styled.form`
```

해당 테그는 form 테그로 변경하는게 좋아보입니다!

Reply:
수정했습니다
[b59ae28](https://github.com/ONEUS-team/gsmatch-front/pull/21/commits/b59ae28de577e850e3b6d263d1f19051999efafb)